docs/notes/pytorch_notes.rst

```diff
@@ -0,0 +1,133 @@
+``PyTorch`` Notes
+************************************
+
+This note is used for doucmenting some useful ``PyTorch``-related features,
+which can serve as a useful reference for future relevent development, either
+implementing algorithms in ``PyTorch`` or converting some eixsting code from
+``TensorFlow`` to ``PyTorch``.
+
+Many of these aspects are encountered during the conversion of ``ALF`` from
+``TensorFlow`` to ``PyTorch``. Some of them are useful features to use, while
```

Comment:
PyTorch, TensorFlow, and ALF are repo names. Probably not quoted as code?

Reply:
I intentionally added the quote to differentiate them from plain text.
I used quote rather than other types of format as it seems to be the most appropriate one.